### PR TITLE
Support selecting organization by invite or creating a new org on first login + fix CLI commands after API changes

### PR DESCRIPTION
**Pull request by amitlicht**
https://www.notion.so/otterize/Weird-behavior-when-logging-in-from-the-cli-first-user-may-remain-org-less-6864fb696e5d43e3b9b983195ea2ff0a
https://www.notion.so/otterize/Remove-active-intents-upload-from-the-mapper-f0da4ba2f74d4db0ab78fb3edfd82207

### CODE REVIEW

File: src/cmd/mapper/mapper-upload.go [hunk 1]
```diff
@@ -21,19 +21,23 @@ var UploadCmd = &cobra.Command{
 			}
 			ctxTimeout, cancel := context.WithTimeout(context.Background(), config.DefaultTimeout)
 			defer cancel()
-			intentsClient := intents.NewClientFromToken(viper.GetString(config.OtterizeAPIAddressKey), config.GetAPIToken(ctxTimeout))
+			intentsClient := graphql.NewClientFromToken(viper.GetString(config.OtterizeAPIAddressKey), config.GetAPIToken(ctxTimeout))
```
Comment (orishoshan) [inline src/cmd/mapper/mapper-upload.go:24]: Can you delete this command? There's a task for it: https://www.notion.so/otterize/Remove-active-intents-upload-from-the-mapper-f0da4ba2f74d4db0ab78fb3edfd82207

File: src/pkg/cloudclient/graphql/genqlient.yaml [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # genqlient config; for full documentation see:
 # https://github.com/Khan/genqlient/blob/main/docs/genqlient.yaml
 schema:
-  - ../../../../cloudgraphql/*.graphql
-  - ../../../../cloudgraphql/federation/*.graphql
+  - ../../../cloudgraphql/*.graphql
+  - ../../../cloudgraphql/federation/*.graphql
 operations:
-- genqlient.graphql
+- ./*.genqlient.graphql
```
Comment (orishoshan) [inline src/pkg/cloudclient/graphql/genqlient.yaml:7]: wut?

Comment (amitlicht) [reply src/pkg/cloudclient/graphql/genqlient.yaml:7]: it workz

File: src/pkg/config/persistent_config.go [hunk 3]
```diff
@@ -16,12 +16,9 @@ import (
 )
 
 type SecretConfig struct {
```
Comment (orishoshan) [inline src/pkg/config/persistent_config.go:18]: rename it

**Review by orishoshan — APPROVED** [review]

**Pull request merged**